docs/admin/logical-replication.rst

```diff
@@ -50,6 +50,13 @@ A publication is created using the
 altered or dropped using corresponding commands. The individual tables can be
 added and removed dynamically using ALTER PUBLICATION.
 
+.. CAUTION::
+
+    Publishing cluster must have
+    :ref:`setting <sql-create-table-soft-deletes-retention-lease-period>`
+    enabled so that subscribing cluster can catch up all changes made during
+    replication pause caused by network issues or explicitly done by a user.
```

Comment:
The soft-delete feature is essential for the replication to work correctly for deleted documents. The lease period should not be larger than the `replication.logical.reads_poll_duration` to get all in-between changes. So its not only related to replication pause.

Reply:
I rephrased a bit - should I also add a sentence about `lease<=poll_duration`, clarifying it a little bit? Smth like if `poll<lease` then poll will see only part of retention

Reply:
the lease should be `>=poll_duration`. a retention lease blocks the removal of (soft) deletes docs from lucene. once deleted docs are removed, the deletion cannot be replicated.

Reply:
thx for the clarification!